lib/crashlogger.ts

```diff
@@ -51,7 +51,7 @@ export function crashlogger(
 		if (!transport) {
 			try {
 				require.resolve('nodemailer');
-			} catch (e) {
+			} catch (e: any) {
```

Comment:
Might as well use [optional catch binding](https://mariusschulz.com/blog/optional-catch-binding-in-typescript).

Reply:
```suggestion
			} catch {
```